sdk/quota/Azure.ResourceManager.Quota/api/Azure.ResourceManager.Quota.netstandard2.0.cs

```diff
@@ -155,17 +140,38 @@ public partial class OperationResponse
         public static bool operator !=(Azure.ResourceManager.Quota.Models.QuotaLimitType left, Azure.ResourceManager.Quota.Models.QuotaLimitType right) { throw null; }
         public override string ToString() { throw null; }
     }
+    public partial class QuotaOperationDisplay
+    {
+        internal QuotaOperationDisplay() { }
+        public string Description { get { throw null; } }
+        public string Operation { get { throw null; } }
+        public string Provider { get { throw null; } }
+        public string Resource { get { throw null; } }
+    }
+    public partial class QuotaOperationResult
+    {
+        internal QuotaOperationResult() { }
+        public Azure.ResourceManager.Quota.Models.QuotaOperationDisplay Display { get { throw null; } }
+        public string Name { get { throw null; } }
+        public string Origin { get { throw null; } }
+    }
     public partial class QuotaProperties
     {
         public QuotaProperties() { }
         public bool? IsQuotaApplicable { get { throw null; } }
         public Azure.ResourceManager.Quota.Models.LimitJsonObject Limit { get { throw null; } set { } }
-        public Azure.ResourceManager.Quota.Models.ResourceName Name { get { throw null; } set { } }
+        public Azure.ResourceManager.Quota.Models.QuotaRequestResourceName Name { get { throw null; } set { } }
         public System.BinaryData Properties { get { throw null; } set { } }
-        public string QuotaPeriod { get { throw null; } }
+        public System.TimeSpan? QuotaPeriod { get { throw null; } }
         public string ResourceType { get { throw null; } set { } }
```

Comment:
This resourceType here is a simple string.
e.g."resourceType": "PublicIPAddresses"

Reply:
Then how about give another name?